app/Models/Campaign.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rogue\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Campaign extends Model
+{
+    use SoftDeletes;
+
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['id', 'internal_title', 'start_date', 'end_date', 'created_at', 'updated_at'];
```

Comment:
Another thing that just occurred to me. Let's not have all these mass-assignable:

```suggestion
    protected $fillable = ['internal_title', 'start_date', 'end_date'];
```

If we want to edit other ones (say, setting a specific ID in an import script), we can do that by manual assignment.